### ISSUE TITLE
Remotely.db size is huge

### ISSUE THREAD

**Issue by itopie** [immense/Remotely#633]
**Describe the bug**

I noticed that the Remotely.db file has become huge (>3.5Go for a very small implementation; <10 remote computers). My first investigations tell me that it could be related to a log issue that would have filled in the db file with too much information.

Another useful information : I cannot access to the logs through the web interface (server logs). The page tries to load but ends up in a timeout / logoff.

**To Reproduce**

I have several instances of Remotely, but only one behaves like this. As I cannot access to the logs, it's difficult for me to understand how to reproduce the bug. 

**Remotely Version**

Version: 2023.04.17.1215
On Debian 11
With Sqlite

**Expected Behavior**

- The Remotely.db file has a reasonable size.
- I can access the server logs

**Desktop**

 - Mixed Windows 10 and Debian 10 
 - Browser : mixed (Firefox and Brave)

**Additional information**

- Do you have a trick to clean up the logs outside the web page? Is there a command for that?
- The appsettings.json (would it be possible to change the loglevel to Error?): 

"Logging": {
    "LogLevel": {
      "Microsoft.AspNetCore.HttpLogging.HttpLoggingMiddleware": "Information",
      "Default": "Information"
    }
  },
  "ApplicationOptions": {
    "AllowApiLogin": false,
    "BannedDevices": [],
    "DataRetentionInDays": 30,
    "DBProvider": "SQLite",
    "EnableWindowsEventLog": false,
    "EnforceAttendedAccess": false,
    "ForceClientHttps": false,
    "KnownProxies": [],
    "MaxConcurrentUpdates": 10,
    "MaxOrganizationCount": 1,
    "MessageOfTheDay": "",
    "RedirectToHttps": true,
    "RemoteControlNotifyUser": true,
    "RemoteControlRequiresAuthentication": true,
    "RemoteControlSessionLimit": 3,
    "Require2FA": true,
    "SmtpDisplayName": "",
    "SmtpEmail": "",
    "SmtpHost": "",
    "SmtpLocalDomain": "",
    "SmtpCheckCertificateRevocation": true,
    "SmtpPassword": "",
    "SmtpPort": 587,
    "SmtpUserName": "",
    "Theme": "Dark",
    "TrustedCorsOrigins": [],
    "UseHsts": false,
    "UseHttpLogging": false
  }

**Comment by lucent-sea** [comment]
Yeah, I noticed this too.  The HTTP logging is overly verbose, and it's being applied regardless of the setting.  That was my mistake.  I assumed it had to be explicitly enabled in app startup, but merely registering the services is enough for it to take effect.

I'll get that changed.  I also wanted to move logging from the DB to flat files via Serilog, so I'll use this as an opportunity to do that.

In the meantime, you can either edit the DB file directly via [DB Browser for SQLite](https://sqlitebrowser.org/).  This is how I do it (from a Windows machine):
- Stop the Docker container.
- Download the DB file to your local machine.
- Create a backup of the file, just in case.
- Open it with DB Browser.
- Go to the EventLogs table, select all, and delete.
- Click "Write Changes" on the top.
- Go to Tools menu, then "Compact Database".
  - Even after deleting the data, the file size won't decrease until this as done.  (As far as I'm aware, anyway.)
- Upload the DB back to the host.

To quiet the logging until it's permanently fixed, you can add this under `LogLevel` in `appsettings.json`:

```
"Microsoft.AspNetCore": "Warning",
```

Then start the container back up.  That should take care of it until there's a permanent fix.

**Comment by itopie** [comment]
Thanks very much for your assistance. I tried the sqlite3 cli tool on my GNU/Linux system.

```
$ sqlite3 Remotely.db
sqlite> .tables
sqlite> SELECT count(*) FROM EventLogs;
9751704
sqlite> DELETE FROM EventLogs;
sqlite> SELECT count(*) FROM EventLogs;
0
sqlite> VACUUM;
```

The last command squeezes the DB file to reclaim storage.

I changed the appsettings.json file. I also had a "Default" directive. I also changed it form "Information" to "Warnings".

After starting the Remotely service, everything went well. The Remotely.db file :
* Before : 3.4Go
* After : 388Ko

:-)

Thanks again.
